app/services/send_apply_to_another_course_when_inactive_email_to_candidate.rb

```diff
@@ -12,4 +13,9 @@ def self.already_sent_to?(application_form)
       chaser_type: :apply_to_another_course_after_30_working_days,
     ).where('created_at > ?', 1.day.ago).present?
   end
+
+  def self.max_application_choice_reached?(application_form)
```

Comment:
How do you feel about having this as a method on the `application_form`?

Reply:
I think it is fine -- I'll make the change, but I'm also concerned about proliferating the number of methods on the form that ostensibly do the same thing. Like how can we have slots available when the number of unsuccessful applications is over 15? I would prefer to do an audit of how we are using those methods more generally.